doc/insx.txt

```diff
@@ -494,6 +494,18 @@ require('insx.recipe.pair_spacing').decrease: fun(option): insx.RecipeSource~
 
 
 
+==============================================================================
+VARIABLE                                                         *insx-variable*
+
+## Buffer Variables
+
+                                             *insx-variable.buffer.insx_disable*
+vim.b.insx_disable: boolean~
```

Comment:
please use `insx_disabled` instead of `insx_disable`.

Reply:
fixed